packages/salesforcedx-vscode-core/src/commands/templates/baseTemplateCommand.ts

```diff
@@ -90,7 +81,7 @@ export abstract class BaseTemplateCommand extends SfdxCommandletExecutor<
       : 'customDir';
   }
 
-  private getPathToSource(outputDir: string, fileName: string): string {
+  public getPathToSource(outputDir: string, fileName: string): string {
```

Comment:
is this necessary?

Reply:
I did this to test something. I will revert it.